tools/travis-test.sh

```diff
@@ -187,6 +193,11 @@ if [ -n "$USE_WHEEL" ] && [ $# -eq 0 ]; then
   pushd dist
   pip install --pre --no-index --upgrade --find-links=. numpy
   pip install nose pytest
+  if [ "$VERSION" == "3.7" ] || [ "$VERSION" == "3.6" ] ; then
+      # use pickle5 backport when on python3.[6-7]
+      pip install pickle5
```

Comment:
Instead of hardcoding the installation of `pickle5` based on the Python version number, please use an explicit environment variable

```bash
if [ "$INSTALL_PICKLE5" == "1" ] ; then
    pip install pickle5
fi
```

and then set that variable on some of the Python 3.6 and/or 3.7 builds in `.travis.yml`.